src/core/server/services/axios.ts/axios.ts

```diff
@@ -0,0 +1,79 @@
+import axios, { Method } from "axios";
```

Comment:
Please move the `axios.ts/axios.ts` to `fetch` instead and export it from `fetch/index.ts`.